packages/deployment/upgrade-test/Makefile

```diff
@@ -9,9 +9,11 @@ endif
 local_sdk:
 	(cd ../ && make docker-build-sdk)
 
+# build main bootstrap
```

Comment:
seems like there's more to say here while we're at it. But the words aren't coming to me. oh well.

maybe borrow from the `$comment` in the .json config files?